app/models/ems_folder.rb

```diff
@@ -118,6 +118,12 @@ def parent_datacenter
     detect_ancestor(:of_type => "EmsFolder") { |a| a.kind_of?(Datacenter) }
   end
 
+  # Indicates if the folder is able to have child VMs
+  def vm_folder?
+    folder_tree = [self] + ancestors.reverse
```

Comment:
There's a method for this already...I think it's called `path`

Reply:
Oh nice, that's a lot cleaner...done